### ISSUE TITLE
PHP 8.2 Deprecated Messages

### ISSUE THREAD

**Issue by Ragowit** [Tustin/Haste#2]
```
[17-Feb-2023 21:06:56 UTC] PHP Deprecated:  Creation of dynamic property Tustin\Haste\Http\JsonStream::$stream is deprecated in /home/psn100/public_html/vendor/guzzlehttp/psr7/src/StreamDecoratorTrait.php on line 21
```

**Comment by Ragowit** [comment]
Oh wait, this might be Guzzle.

**Comment by Ragowit** [comment]
Not a Guzzle issue, it's within Haste.

https://github.com/guzzle/psr7/issues/549#issuecomment-1435356202

**Comment by Ragowit** [comment]
This is a very frequent error and my error_log is expanding quite fast. I would like this to have a higher priority.

**Comment by Tustin** [comment]
Sorry @Ragowit I've been busy and I'm just now catching this... please let me know if this PR works for you 😄

**Comment by Ragowit** [comment]
The PR works great, thanks!

**Comment by Ragowit** [comment]
Fixed with c3fe6eb28d45626356062e248a77e29d0d01b346
